### PR TITLE
Home page styling

### PR DESCRIPTION
**Pull request by alikew24**
### What is a quick description of the change?

Style home page to look more welcoming, add an about section and a meet the creators section. 

### Is this fixing an issue?

none

### Are there more details that are relevant?

none

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [  ] Tests passing
- [ x ] Coding style (indentation, etc)

Please explain why any are not present, if any.

no need to test just html code

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -3,14 +3,83 @@
   <head>
     <meta charset="UTF-8">
     <title>Friends of Friends</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+  </head>
```
Comment (jessdejong) [inline src/main/webapp/index.html:12]: nit: two head tags?

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -3,14 +3,83 @@
...
+        </nav>
+        <div class="home-page-content-header">
+          <h1>Friends of Friends</h1>
+          <img src="images/logo.jpg" alt="logo" class="rounded-circle image-fluid" height="100" width="120">
+        </div>
+        <img src="images/friends-of-friends.png" class="image-fluid" alt="home page image">
+        <div class="home-page-content" id="about">
+          <h2>About</h2>
+          <p>
+            Friends of Friends is a friend finding app! Simply log in with Facebook, and we will look at your friends lists and the friends lists of other users on the app
```
Comment (jessdejong) [inline src/main/webapp/index.html:42]: nit: if it doesn't mess up the formatting, line wrapping these lines a bit more probably would be better

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -1,64 +1,107 @@
...
-    background-color: #fbe8a6;
+  background-color: #fbe8a6;
 }
 .card-container:nth-child(odd) {
-    background-color: #fbe8a6;
-    justify-content: center;
+  background-color: #fbe8a6;
+  justify-content: center;
 }
 .card-container:nth-child(odd) a{
```
Comment (jessdejong) [inline src/main/webapp/style.css:96]: nit: space after a

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -3,14 +3,86 @@
...
+                <td>Isabelle was in charge of the backend algorithms for Friends of Friends. She worked on using Maps to determine which users 
+                    were potential matches for each other by comparing users Facebook Friends Lists. She also worked on determining who matches with who based 
+                    on who mutually chooses "Friend Me" on each other.She is a rising junior at Georgia Tech.  
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div> <!--content-->
+    </div> <!--container-fluid-->
```
Comment (estep-google) [inline src/main/webapp/index.html:85]: Perfect use of end tag comments!

File: src/main/webapp/index.html [hunk 5]
```diff
@@ -3,14 +3,86 @@
...
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td><img src="images/jess.jpg" class="image-fluid" alt="home page image" height="200" width="200"></td>
+                <td><img src="images/alisha.jpg" class="image-fluid" alt="home page image" height="200" width="200"></td>
+                <td><img src="images/isabelle.jpg" class="image-fluid" alt="home page image" height="200" width="200"></td>
+              </tr>
+              <tr>
+                <td>Jess worked on the backend of Friends of Friends. His work allows  users to log in with their facebook account, store their information in datastore, and
```
Comment (estep-google) [inline src/main/webapp/index.html:69]: I would expect text/content to nest the same way it does in a paragraph tag:

```
<td>
  Hi content!
</td>
```

**Comment by alikew24** [comment]
addressed these comments!

File: src/main/webapp/index.html [hunk 6]
```diff
@@ -66,16 +66,19 @@ <h2>Meet the Creators</h2>
...
-                    includes using javascript to pre-populate the profile, feed, and matches pages to be targeted to the current logged in user. She is a 
-                    rising Information Science and Systems Technology major at Cornell University.  </td>
-                <td>Isabelle was in charge of the backend algorithms for Friends of Friends. She worked on using Maps to determine which users 
-                    were potential matches for each other by comparing users Facebook Friends Lists. She also worked on determining who matches with who based 
-                    on who mutually chooses "Friend Me" on each other.She is a rising junior at Georgia Tech.  
+                <td>
+                  Alisha worked on the frontend of the application and designed the user interface using HTML, CSS, and Javascript. Her work also 
+                  includes using javascript to pre-populate the profile, feed, and matches pages to be targeted to the current logged in user. She is a 
+                  rising Information Science and Systems Technology major at Cornell University.  </td>
```
Comment (estep-google) [inline src/main/webapp/index.html:77]: nit: newline for the `</td>`

**Pull request merged**